params/a_voyerii.R

```diff
@@ -0,0 +1,14 @@
+# input parameters
+# ________________________________________________
+
+
+# sigma2: 0 < value < 5
+sig2 <- 4.2 
+    
+# e.g. "anas_krystallinus"    
+species.name <- ale_voyerii
```

Comment:
Sorry Alejandro, Species name needs to be a character vector, Can you please fix?